users/vieting/experiments/switchboard/hybrid/feat/experiments.py

```diff
@@ -208,8 +208,31 @@ def run_baseline_scf():
         # noinspection PyUnresolvedReferences
         train_job.rqmt.update({"gpu_mem": 24, "mem": 10})
 
+def run_specaug_scf():
+    gs.ALIAS_AND_OUTPUT_SUBDIR = "experiments/switchboard/hybrid/feat/"
+
+    nn_args = get_nn_args_baseline(
+        nn_base_args={
+            "scf": dict(
+                returnn_args=dict(batch_size=7000, specaug_mask_sorting=True),
+                feature_args={"class": "ScfNetwork", "size_tf": 256 // 2, "stride_tf": 10 // 2}
+            )
+        },
+        prefix="conformer_bs14k_specaug_sorted",
```

Comment:
```suggestion
        prefix="conformer_bs14k_specaug_sorted_",
```
Otherwise we get an alias `"...sortedscf"`